commons/src/main/java/com/connexta/replication/spring/ReplicationProperties.java

```diff
@@ -0,0 +1,50 @@
+package com.connexta.replication.spring;
+
+import java.util.concurrent.TimeUnit;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties(prefix = "replication")
+public class ReplicationProperties {
+
+  private long period;
+
+  private int connectionTimeout;
+
+  private int receiveTimeout;
+
+  public long getPeriod() {
+    return period;
+  }
+
+  public void setPeriod(long period) {
+    this.period = period;
+  }
+
+  // We let the user set the connection timeout in seconds for readability but most clients take
+  // timeouts in milliseconds so we'll go ahead and do the conversion for them here.
+  public int getConnectionTimeout() {
+    if (connectionTimeout > 0) {
```

Comment:
I should maybe test this code but it's not complex and I don't want the drop in code coverage caused by having jacoco analyze this module full of pojos.